tests/test_getlocal.F90

```diff
@@ -5,37 +5,153 @@
 program test_getlocal
     implicit none
```

Comment:
In future please add your name for any test you author or significantly add to.

Also add the date created if you create the test.

Reply:
Thank you, I'll remember to do that next time.